### ISSUE TITLE
Does gramine support EDMM feature?

### ISSUE THREAD

**Issue by bronzeMe** [gramineproject/gramine#665]
## Description of the problem

* Gramine usually take a long time to load an application into enclave, which provably due to the high overhead of creating or adding EPC page.
* So, EDMM feature of SGX 2.0 may be a solution to reduce the overhead of loading a gramine-sgx application.
* The questions are:
* Q1: Does gramine support EDMM feature? and how do we enable it?
* Q2: Is there other nice optimization solution for  gramine?
Thanks in advance!

**Comment by mkow** [comment]
No, but we're working on this actively (mostly @vijaydhanraj). The biggest blocker right now is that EDMM support was not upstreamed to Linux yet.

**Comment by mkow** [comment]
Umm, this is a _driver_. You can't "invoke" it without having it already inside the kernel.
And also, the driver you linked to was deprecated long time ago.

**Comment by bronzeMe** [comment]
Thanks, your comments are helpful.
